Exercise_1.java

```diff
@@ -5,29 +5,53 @@
   
     boolean isEmpty() 
     { 
-        //Write your code here 
+        if(top == -1){

```

Comment:
For reference purposes:
This if else block can be written in one line as well return top < 0 ? true : false;
or even better would be to write return top == -1; since top cannot go lower than -1